docs/tutorial/kymographs.rst

```diff
@@ -140,6 +140,48 @@ This returns a new (but flipped) :class:`~lumicks.pylake.kymo.Kymo`::
 
 .. image:: figures/kymographs/kymo_flipped.png
 
+Extracting photon counts
+------------------------
+
+Photon counts can be extracted from the raw image, for example for stoichiometry analysis.
+First, select a region for which the photon counts have to be determined::
+
+    kymo_selection = kymo["88.5s":"93s"].crop_by_distance(21.4, 22.3)
+
+    plt.figure()
+    kymo_selection.plot("rgb", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+
+Get the raw data (photon counts for each pixel) for the selected region::
+
+    selection_green = kymo_selection.get_image(channel="green")
+
+.. image:: figures/kymographs/kymograph_selection.png
+
+If the background has to be subtracted, choose a region without binding events as background::
+
+    background =  kymo["92.1s":"100s"].crop_by_distance(21.4, 22.3)
+    
+    plt.figure()
+    background.plot("rgb", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+
+.. image:: figures/kymographs/background.png
+
+Sum over the vertical axis (along the width of the binding event) to obtain the total photon counts over the selected regions.
+Since every pixel has an average background signal of `background_mean`, we need to sum the background over the same region to obtain the total background.
```

Comment:
```suggestion
Since every pixel has an average background signal of `background_mean`, we need to sum the background over the same region to obtain the total background::
```